plutus-ledger-api/src/PlutusLedgerApi/V3/Contexts.hs

```diff
@@ -407,6 +413,26 @@ data GovernanceAction
   deriving anyclass (HasBlueprintDefinition)
   deriving (Pretty) via (PrettyShow GovernanceAction)
 
+instance PlutusTx.Eq GovernanceAction where
+  {-# INLINABLE (==) #-}
+  ParameterChange govId cp sh == ParameterChange govId' cp' sh' =
+    govId PlutusTx.== govId' PlutusTx.&& cp PlutusTx.== cp' PlutusTx.&& sh PlutusTx.== sh'
+  HardForkInitiation govId vers == HardForkInitiation govId' vers' =
+    govId PlutusTx.== govId' PlutusTx.&& vers PlutusTx.== vers'
+  TreasuryWithdrawals wthdrl sh == TreasuryWithdrawals wthdrl' sh' =
+    wthdrl PlutusTx.== wthdrl' PlutusTx.&& sh PlutusTx.== sh'
+  NoConfidence govId == NoConfidence govId' =
+    govId PlutusTx.== govId'
+  UpdateCommittee govId toDel toAdd quorum == UpdateCommittee govId' toDel' toAdd' quorum' =
+    govId PlutusTx.== govId'
+      PlutusTx.&& List.sort (List.nub toDel) PlutusTx.== List.sort (List.nub toDel')
```

Comment:
This is also problematic - I imagine there are cases where you just want the plain `==`.